app/js/components/subAddress/addressForm.jsx

```diff
@@ -288,26 +288,30 @@ export default class AddressForm extends React.Component {
                   </div>
                   {this.state.action === "edit" ?
                     <div className="form-group">
-                      <div className="col-sm-4">
-                        <button type="button" name="cancel" onClick={this.cancelClick}
-                          className="btn btn-danger form-control cancelBtn">Cancel</button>
-                      </div>
-                      <div className="col-sm-4">
-                        <button type="button" name="update" onClick={(e) => this.update(e, this.state.uuid)}
-                          className="btn btn-success form-control">Update</button>
+                      <div id="buttons">
+                        <div className="col-sm-4">
+                          <button type="button" name="cancel" onClick={this.cancelClick}
+                            className="btn btn-danger form-control cancelBtn">Cancel</button>
+                        </div>
+                        <div className="col-sm-4">
+                          <button type="button" name="update" onClick={(e) => this.update(e, this.state.uuid)}
+                            className="btn btn-success form-control">Update</button>
+                        </div>
                       </div>
                     </div>
                     : ''
                   }
                   {this.state.action === "new" ?
```

Comment:
You seem to be using the action state in multiple places. Destructuring the state can make for more clean and readable code. e.g. `const { action, uuid, ...otherInterestingStates } = this.state`